app/controllers/concerns/blacklight/search_context.rb

```diff
@@ -78,6 +78,13 @@ def agent_is_crawler?
     crawler_proc.call(request)
   end
 
+  def skip_session_tracking?
```

Comment:
I haven't look at what `track_search_session` is, but what if we let it be a boolean and/or a lambda?

Reply:
Yeah that might be a better extendable approach. I'll look into that.